### ISSUE TITLE
About the monotonicity of the predict layer in the coral network

### ISSUE THREAD

**Issue by Tony-958** [Raschka-research-group/coral-cnn#30]
Hi,
Sorry to bother. I had a problem when understanding the monotonicity of the coral network's predict layer. Here's the only statement I found modifying the bias layer adding to the fc result:
self.linear_1_bias = nn.Parameter(torch.zeros(self.num_classes-1).float())
Is it enough to restrict the monotonicity ? Or is there any other statement restricting the  monotonicity of the biases？

**Comment by rasbt** [comment]
I would call it "guaranteeing" monotonicity :). Yeah, it's the bias and the weight sharing together with the loss function. Regarding the changes compared to a regular network, maybe this overview helps? https://github.com/Raschka-research-group/coral-cnn/blob/master/github-images/differences-at-a-glance.pdf

**Comment by Tony-958** [comment]
Yeah, that helps. Thank you so much.
